CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## v5.15.0
```

Comment:
keep unreleased beside this. If this is a patch, then version should be v5.14.1

Reply:
We support new resources in this release. So this is minor release, not a patch